pages/index.js

```diff
@@ -292,9 +290,29 @@ export default function Home() {
       >
         {!isLoading &&
           (riteBalance > 0 ? (
-            <RiteStaked balance={riteBalance} deadline={stakeDeadline} />
+            <RiteStaked
+              setDisplaySponsorCohort={setDisplaySponsorCohort}
+              displaySponsorCohort={displaySponsorCohort}
+              balance={riteBalance}
+              deadline={stakeDeadline}
+              minimumStake={minimumStake}
+              context={context}
+              raidBalance={raidBalance}
+              allowance={allowance}
+              isChecked={isChecked}
+              handleIsChecked={handleIsChecked}
+              cohortAddress={cohortAddress}
+              handleCohortAddress={handleCohortAddress}
+              isApproveTxPending={isApproveTxPending}
+              makeAnAllowance={makeAnAllowance}
```

Comment:
These components have an awful lot of props, perhaps some of these could be added to a context?